string serial_id
geometry_msgs/Pose2D goal_pose
---
geometry_msgs/Point[] path
